modules/adding-a-custom-logo.adoc

```diff
@@ -11,7 +11,7 @@ You can create custom branding by adding a custom logo or custom product name. Y
 .Prerequisites
 
 * You must have administrator privileges.
-* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-width` of `200px` and a max-height` of `68px`. Image size must not exceed 1 MB due to constraints on the `ConfigMap` object size.
+* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a max-height` of `60px`. Image size must not exceed 1 MB due to constraints on the `ConfigMap` object size.
```

Comment:
```suggestion
* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a max-height of `60px`. Image size must not exceed 1 MB due to constraints on the `ConfigMap` object size.
```

Reply:
I actually think it should be in backticks. I have updated to fix the typo!